src/main/java/org/jboss/forge/arquillian/command/algeron/consumer/PactAlgeronConsumerTestSetup.java

```diff
@@ -33,6 +33,7 @@ public JavaClassSource updateTest(JavaClassSource test, String consumer, String
         annotateTestClass(test, consumer, provider);
         createContractFragment(test, fragmentName);
         createTestMethod(test, consumer, provider, fragmentName);
+        removeTestMethod("should_be_deployed", test);
```

Comment:
Could we find if there is an static method annotated with `@Deployment` and if it is the case remove it? This is what we want to do it isn't it? And it is in a generic way.